pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -290,6 +290,7 @@ export const ApplicationForm: FC<ApplicationFormProps> = memo(
             <TextField
               id="repoPath"
               label="Path"
+              placeholder="Path (use ./ as repo root)"
```

Comment:
Can you tell me the relation between this change that the linked issue?

Reply:
Sorry for the lack of specific explanation.
The back story was user is unable to know how should they use for `Path` field in Add Application form (which a required field). The root cause of this issue is this line in proto file
```proto
string path = 2 [(validate.rules).string.pattern = "^[^/].+$"];
```
ref: https://github.com/pipe-cd/pipe/blob/master/pkg/model/common.proto#L40
I think we have serval ways to fix this
- disable this validation
- mark Path field in form as not required
- notice users that they can use `./` as repo root (as I do in this PR)
We use this path in serval places such as load the deployment configuration, build the configuration directory info for application detail, plan-preview repo root config, etc. With the current number of being used places, I think having validation would still be better (it avoids case when `//` - double slash appeared in the built path from our logic). Besides, I realized in case `.` appeared in the URL, it will be omitted by default (no hurt the URL building logic) and for git tasks, or normal file retrievement the `.` should be fine too.

Reply:
Thank you for your explanation. I got your points.
I just feel that this placeholder value is hard to understand.
It would be nice if we could have a better solution (better text or better way). For example:
- change the placeholder to `./ means repository root`
- or set that content with a longer text in HelperText field instead of a placeholder
- or set default value to repo root

Reply:
I see, got your point. 👍 
>  set default value to repo root

I think it would be better to keep this field as a required field so that users will be aware of their settings. Will adopt the HelperText field solution.

Reply:
updated, PTAL 🙏